OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -284,6 +284,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface ITemporaryBlocker
```

Comment:
Could we get a `[RequireExplicitImplementation]` here too?  Now that we have that interface we may as well use it.